ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -67,13 +67,18 @@ class FaultInjector
   
 private:
   float powerTemperatureOverloadValue;
+  float powerStateOfChargeValue;
+  float originalSOC;
+  // float powerStateOfChargeValue;
   
   // renders new temperature when thermal power fault is re-triggered
-  void setPowerTemperatureFaultValue(bool getTempBool);
+  float getRandomValueFromRange(float min_val, float max_val);
```

Comment:
Minor: you could rename this function for the probability distribution it uses, if that's knowable and this is useful.

Reply:
gotcha! will do